source/distributor/service_impl.cc

```diff
@@ -0,0 +1,117 @@
+#include "distributor/service_impl.h"
+
+#include <grpc++/grpc++.h>
+
+#include "envoy/config/core/v3/base.pb.h"
+
+#include "external/envoy/source/common/common/assert.h"
+#include "external/envoy/source/common/protobuf/message_validator_impl.h"
+#include "external/envoy/source/common/protobuf/utility.h"
+
+#include "api/distributor/distributor.pb.validate.h"
+
+namespace Nighthawk {
+
+::grpc::Status NighthawkDistributorServiceImpl::validateRequest(
+    const ::nighthawk::DistributedRequest& request) const {
+  auto& validation_visitor = Envoy::ProtobufMessage::getStrictValidationVisitor();
```

Comment:
nit: I think this would be more readable with ValidationVisitor& here instead of auto&.  The type inference isn't quite obvious enough for me.

Reply:
done in 6471b8687332bcae77a637c8b1d86640299e20c6